### DISCUSSION TITLE
thrift.apache  fixes the vulnerabilities

### DISCUSSION THREAD

**Discussion by hanyaohui123** [community/community#119755] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://thrift.apache.org/   Does the software involve the CVE-2019-11938 and CVE-2019-3553 vulnerabilities? Which version fixes the vulnerabilities?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @hanyaohui123 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Feed category is solely related to conversations around the GitHub product Feed. This question should be in the General category. I've gone ahead and moved it for you. Good luck!
